### DISCUSSION TITLE
如何上传本地文件到网盘？

### DISCUSSION THREAD

**Discussion by liaojiaxin97** [AlistGo/alist#7359] [Q&A]
已经挂载谷歌云盘，如何将本地文件上传？若能上传速度会比正常快吗？

**Comment by ririye** [comment]
![QQ截图20241019142759](https://github.com/user-attachments/assets/2dbf1194-c04c-4e6c-a3e9-e403c514a5c8)
这里上传，速度一样的，甚至挂载aliast的机器不好的话，还不如网盘直接上传

**Comment by pongfcnkl** [comment]
谷歌得挂代理吧
先添加本机存储，然后用网页上的复制功能上传

**Comment by snugduahn** [comment]
你的是速度慢吗？我的是直接不可以上传，一上传说报错。
